cpg_infra/billing_aggregator/aggregate/utils.py

```diff
@@ -797,15 +800,50 @@ def get_unit_for_batch_resource_type(batch_resource_type: str) -> str:
 
 
 def get_start_and_end_from_request(
-    request,
+    request: Request,
 ) -> tuple[datetime | None, datetime | None]:
     """
     Get the start and end times from the cloud function request.
     """
-    if request:
-        print(request)
-        # return request.params['start'], request.params['end']
-    return None, None
+    if not request:
+        return None, None
+
+    content_type = request.content_type
+    if request.method == 'GET':
+        print('GET request, using args')
```

Comment:
Can you switch the prints to logging, also could be worth logging the args in these blocks below too